Python/errors.c

```diff
@@ -126,6 +126,7 @@ _PyErr_GetTopmostException(PyThreadState *tstate)
     {
         exc_info = exc_info->previous_item;
     }
+    assert(!Py_IsNone(exc_info->exc_value));
```

Comment:
Can we now change the while loop above to skip over only nulls?

Reply:
Yes, sorry, forgot to extricate that bit from the other PR. Done now.